datafusion/physical-plan/src/lib.rs

```diff
@@ -428,6 +428,18 @@ pub trait ExecutionPlan: Debug + DisplayAs + Send + Sync {
     fn statistics(&self) -> Result<Statistics> {
         Ok(Statistics::new_unknown(&self.schema()))
     }
+
+    /// Returns `true` if a limit can be safely pushed down through this
+    /// `ExecutionPlan` node.
```

Comment:
I think it might help to also explain the implications of returning true. Something like

```suggestion
    /// `ExecutionPlan` node.
    ///
    /// If this method returns true, and the query contains a `FETCH`/`LIMIT`
    /// at the output of this node, DataFusion will push the Fetch/Limit to 
    /// the input of this node
```